specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2018-08-01/resourcehealth.json

```diff
@@ -0,0 +1,772 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-08-01",
```

Comment:
If this is a preview api-version, please add a suffix to the apiversion to indicate that.

Reply:
Done